bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "px-vis-timeseries",
-  "version": "4.0.1",
+  "version": "4.1.0",
```

Comment:
Why is this a minor bump? No features have been added. 

As this changes nothing user or developer facing (ie. someone who has px-vis-timeseries as a project dependency), I don’t think a new version should be released.

Reply:
@evanjd figured it wasn't a patch either. fair enough-